extraTS/utils/helpers.ts

```diff
@@ -0,0 +1,10 @@
+export default function validateEmail(email){
+    const regex = /\w+.\w+@successive.tech$/i;
+    if(regex.test(email))
+    {
+        return true;
+    }
+    else{
+        return false;
+    }
+}
```

Comment:
Write the return type of function

Reply:
I make the function as per given in the task but by mistaken make wrong the folder .i resolved it.